ui/plugin-system/src/components/Variables/variable-model.ts

```diff
@@ -39,11 +39,14 @@ export function filterVariableList(data: VariableOption[], capturedRegexp: RegEx
   return result;
 }
 
-export function useListVariablePluginValues(definition: ListVariableDefinition): UseQueryResult<VariableOption[]> {
+export function useListVariablePluginValues(
+  definition: ListVariableDefinition,
+  refreshToken?: string
+): UseQueryResult<VariableOption[]> {
   const { data: variablePlugin } = usePlugin('Variable', definition.spec.plugin.kind);
   const datasourceStore = useDatasourceStore();
   const allVariables = useAllVariableValues();
-  const { absoluteTimeRange: timeRange, refreshKey } = useTimeRange();
```

Comment:
why do we remove this? wouldn't be better to actually check why this refreshKey is not being set with a value that can invalidate cache?

Reply:
ok, thnx for clarifying. In that case I wonder if we need this refreshKey at all. Shouldn't the actual timeRange change as it uses the current time already? This will effectively invalidate the cache for a selected time range, without causing rendering loops. In the case of variable editor I'd assume the current time minus a predefined range is used.